static_src/components/breadcrumbs.jsx

```diff
@@ -51,7 +51,7 @@ export default class Breadcrumbs extends React.Component {
 
     // Add the parent states
     if (org && org.name) {
-      breadcrumbs.push(<BreadcrumbsItem key="home" url="/#/">overview</BreadcrumbsItem>);
+      breadcrumbs.push(<BreadcrumbsItem key="home" url="/#/">Overview</BreadcrumbsItem>);
```

Comment:
I just wanted to make sure this change was intentional because at one point it was explicitly lowercased. I'm a fan of consistency and this looks okay to me.

![screenshot from 2017-01-12 15-23-43](https://cloud.githubusercontent.com/assets/509703/21912506/458234c6-d8db-11e6-8427-987a88a59d94.png)